### DISCUSSION TITLE
Paranoid vs Soft Deletion

### DISCUSSION THREAD

**Discussion by mikizdr** [sequelize/sequelize#17235] [Help & Questions]
Why is the "paranoid" term used for soft deletion of the records, and just not softdelete or softDelete? I don't see and don't understand the naming convention here that should be intuitive and understandable. And at the same time, I see a confusable and non-understandable entity in the system.

**Comment by ephys** [comment]
The name probably comes from other ORMs, can't know for sure. It was named that way [13 years ago](https://github.com/sequelize/sequelize/commit/2549100527cf4cea7635b53c417399dc23ff3fdf)

I personally would prefer if it were named "softDelete" and "hardDelete" respectively, as that name is a lot more standard than "paranoid" (plus I don't think we should use the name of medical conditions)

Deprecating the old name & adding support for a new one is not super high on the priority list however
